framework/src/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -1324,6 +1326,58 @@ private void addAnnotationToStore(CFStore store, AnnotationMirror anno, Node nod
                 transferResult, firstNode, secondNode, firstValue, secondValue, notEqualTo);
     }
 
+    @Override
+    protected void processConditionalPostconditions(
+            MethodInvocationNode n,
+            ExecutableElement methodElement,
+            Tree tree,
+            CFStore thenStore,
+            CFStore elseStore) {
+        // For String.startsWith(String) and String.endsWith(String), refine the minimum length
+        // of the receiver to the minimum length of the argument.
+        // Basically, MinLen annotations are transferred from the argument to the receiver.
+        // However, a polymorphic qualifier cannot be used to express this, because
```

Comment:
I added this comment on this request: https://github.com/panacekcz/checker-framework/pull/6#pullrequestreview-52566329. I did not investigate this in detail because the reason stated in this comment was enough to tell that the polymorphic qualifier is not a solution. If it cannot work in principle, then I don't know why it was proposed in the first place.